server/src/models/database/validatable.ts

```diff
@@ -0,0 +1,8 @@
+export interface Validatable {
```

Comment:
there is no implementation of this interface. i like interfaces prefixed with an _I_. I don't know why this is bad!?